.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Libra Developers Website CI
+
+on:
+  push:
```

Comment:
Do we need `push` here? Or would `pull_request` just work? I noticed in this PR that we have two GHA status checks, both look the same to me.

Reply:
I'll restrict it to the auto branch (which is needed for bors)